src/pages/CompletedSurvey/ui/CompletedSurveyPage.module.scss

```
@@ -0,0 +1,50 @@
+@import 'src/app/styles/variables/global.scss';
+
+.CompletedSurveyPage {
+    padding-top: 21px;
+    display: flex;
+    gap: 22px;
+    flex-direction: column;
+    align-items: center;
+}
+
+.feedbackBlock {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 28px;
```

у этого блока есть `width` для десктопа и таблета - `550px`
для мобилки - нет, там `width: 100%`
```suggestion
    gap: 28px;
    width: 550px;
    max-width: 550px;
```